bears/python/PyFlakesBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='pyflakes',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
+                     r'(?P<column>\d*):*(?P<severity>)\s(?P<message>.*)\n',
+        severity_map={
+            '': RESULT_SEVERITY.INFO
+        })
```

Comment:
@jayvdb Wants all the errors to be addressed as INFO. But by default all the errors are logged to NORMAL.